system-addon/data/content/tippytop/top_sites.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "title": "amazon",
+    "urls": ["https://www.amazon.com/", "https://www.amazon.ca/", "https://www.amazon.de/", "https://www.amazon.co.uk/", "https://www.amazon.fr/"],
+    "image_url": "amazon-com@2x.png"
+  },
+  {
+    "title": "facebook",
+    "url": "https://www.facebook.com/",
+    "image_url": "facebook-com@2x.png"
```

Comment:
It seems reasonable to access it directly here, but I wonder if we need to solve this issue more broadly for things down the line? adding it to `store.App`makes sense, although ordering would still be an issue I guess